Sources/Plasma/FeatureLib/pfPython/cyAvatar.cpp

```diff
@@ -1614,6 +1614,33 @@ bool cyAvatar::LoadClothingFromFile(plFileName filename)
     return false;
 }
 
+/////////////////////////////////////////////////////////////////////////////
+//
+//  Function   : FindBone
+//  PARAMETERS : bonename - the name of the bone to find
+//
+//  PURPOSE    : Returns a reference to the scene object for the bone with the
+//               given name.
+//
+PyObject* cyAvatar::FindBone(const ST::string& boneName)
+{
+    // make sure that there is atleast one avatar scene object attached (should be)
+    if (!fRecvr.empty()) {
+        // find the armature modifier
+        const plArmatureMod* avatar = IFindArmatureMod(fRecvr[0]);
+        if (avatar) {
+            const plSceneObject* bone = avatar->FindBone(boneName);
+            if (bone) {
+                return pySceneObject::New(bone->GetKey());
+            }
+        }
+    }
+
+    ST::string errmsg = ST::format("Bone {} not found", boneName);
+    PyErr_SetString(PyExc_NameError, errmsg.c_str());
```

Comment:
```suggestion
    PyErr_SetString(PyExc_ValueError, errmsg.c_str());
```

This is a little pedantic, but [`NameError`](https://docs.python.org/3/library/exceptions.html#NameError) is raised for global or local variable names that don't exist.